addons/web/static/tests/views/calendar_tests.js

```diff
@@ -1271,6 +1269,59 @@ QUnit.module('Views', {
         calendar.destroy();
     });
 
+    QUnit.test('create event with default context (no quickCreate)', async function (assert) {
+        assert.expect(1);
+
+        this.data.event.records = [];
+
+        var calendar = await createCalendarView({
+            View: CalendarView,
+            model: 'event',
+            data: this.data,
+            arch:
+                '<calendar class="o_calendar_test" ' +
+                'date_start="start" ' +
+                'date_stop="stop" ' +
+                'mode="week" ' +
+                'quick_add="False"/>',
+            archs: archs,
+            viewOptions: {
+                initialDate: initialDate,
+            },
+            session: {
+                getTZOffset: function () {
+                    return 120;
+                },
+            },
+            context: {
+                default_name: 'New',
+            },
+            intercepts: {
+                do_action: function (event) {
+                    assert.deepEqual(event.data.action.context, {
+                        "default_name": "New",
+                        "default_start": "2016-12-14 05:00:00",
+                        "default_stop": "2016-12-15 17:00:00",
+                    },
+                    "should send the correct data to create events");
+                },
+            },
+        }, { positionalClicks: true });
+
+        var pos = calendar.$('.fc-bg td:eq(4)').offset();
+        try {
+            testUtils.dom.triggerPositionalMouseEvent(pos.left + 15, pos.top + 15, "mousedown");
+        } catch (e) {
+            calendar.destroy();
+            throw new Error('The test fails to simulate a click in the screen. Your screen is probably too small or your dev tools is open.');
+        }
+        pos = calendar.$('.fc-bg td:eq(5)').offset();
+        testUtils.dom.triggerPositionalMouseEvent(pos.left + 15, pos.top + 15, "mousemove");
+        testUtils.dom.triggerPositionalMouseEvent(pos.left + 15, pos.top + 15, "mouseup");
```

Comment:
This triggers the `do_action` when this works as intended. Should see an `assert.verifySteps()` with step from the intercepted `do_action`